### PR TITLE
Delete book and its relationship

### PR DESCRIPTION
**Pull request by mvn-minhphan-dn**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Product Backlog ID: PBI-8
- Task ID: TI-3

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

- resources/views/backend/books/list.blade.php
- resources/views/backend/layouts/main.blade.php
- routes/web.php

### CODE REVIEW

File: app/Http/Controllers/Admin/BookController.php [hunk 1]
```diff
@@ -202,4 +203,20 @@ public function update(BookEditRequest $request, Book $book)
...
+     * @param Request $request request
+     * @param int     $id      id
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy(Request $request, $id)
+    {
+        $book = Book::find($id)->delete();
+        if ($request->ajax()) {
+            return response()->json(['book'=> $book], 200);
```
Comment (at-anhngo) [inline app/Http/Controllers/Admin/BookController.php]: chỉ cần return response()->json($book, 200) là được

File: public/js/delete_list.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+			if ($data.book.id == undefined){
+				$tr = $('#' + $id).parent().parent();
+				$tr.attr('style', 'background-color: #aaa');
+				setTimeout(function(){
+					$tr.remove();
+				}, 1000);
+			}
+		}
+	});
+});
```
Comment (at-anhngo) [inline public/js/delete_list.js]: lỗi frontend 4 spaces

**Review by ghost — CHANGES_REQUESTED** [review]
Check my comment

File: app/Http/Controllers/Admin/BookController.php (hunk 1 above)
Comment (at-anhngo) [reply app/Http/Controllers/Admin/BookController.php]: nếu xử lý ajax thì bỏ trong app/Http/Controllers/Api

File: public/js/delete_list.js (hunk 2 above)
Comment (at-annguyen) [reply public/js/delete_list.js]: js là 2 spaces hay 4 spaces????

File: app/Http/Controllers/Admin/BookController.php (hunk 1 above)
Comment (at-annguyen) [reply app/Http/Controllers/Admin/BookController.php]: đã hỏi anh Việt và không cần thiết bỏ trong app/Http/Controllers/Api

**Review by ghost — APPROVED** [review]

**Comment by at-anhngo** [comment]
LGTM

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**